component/vdr/cheqd/vdr.go

```diff
@@ -0,0 +1,245 @@
+package cheqd
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/hyperledger/aries-framework-go/pkg/common/log"
+	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
+
+	diddoc "github.com/hyperledger/aries-framework-go/pkg/doc/did"
+	vdrapi "github.com/hyperledger/aries-framework-go/pkg/framework/aries/api/vdr"
+)
+
+var logger = log.New("aries-framework-go-ext/vdr/cheqd")
+
+// VDR implements the VDR interface.
+type VDR struct {
+	endpointURL      string
+	client           *http.Client
+	accept           Accept
+	resolveAuthToken string
+}
+
+// Accept is method to accept did method.
+type Accept func(method string) bool
+
+// Option configures the peer vdr.
+type Option func(opts *VDR)
+
+// New creates new DID Resolver.
+func New(endpointURL string, opts ...Option) (*VDR, error) {
+	v := &VDR{client: &http.Client{}, accept: func(method string) bool { return true }}
```

Comment:
Is it expected that it always returns true? I think it should return true for cheqd method only, shouldn't it?

Reply:
What's the reason to use that stub? Why don't we provide a correct default value:
`func(method string) bool { return method == "cheqd" }`? This method is supposed to be called for cheqd only, so doesn't make sense to specify anything else.

Reply:
Done! ✅